tools/clang/test/DXC/old_pdb.test

```diff
@@ -0,0 +1,7 @@
+// RUN: %dxc /dumpbin %S/Inputs/old_pdb.pdb | FileCheck %s
+
+// CHECK: define void @main
```

Comment:
We really want to move away from tests that just test for `@main`. Could you check the sizes or something?

Reply:
The only thing we want to test is the ability to read the PDB. If we read the DXIL module out of the PDB, we've successfully done that. I don't think we have any command-line utilities that do any PDB specific things that we can run.